CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.1.6
+  - Add error log backtrace [#54](https://github.com/logstash-plugins/logstash-filter-ruby/pull/54)
```

Comment:
Clarifying the scope of the new logging, as it influences only inline ruby scripts but not file base scripts.

```suggestion
  - Add error log backtrace to inline scripts [#54](https://github.com/logstash-plugins/logstash-filter-ruby/pull/54)
```